.github/workflows/label-pr.yml

```diff
@@ -64,21 +67,23 @@ jobs:
           type: remove
 
   pr_review:
-    permissions:
-      pull-requests: write
     runs-on: ubuntu-latest
     needs: job_picker
     if: needs.job_picker.outputs.run == 'pr_review'
     steps:
-      - name: Remove needs-review label
-        if: >
-          github.event.review.author_association == 'COLLABORATOR' ||
-          github.event.review.author_association == 'OWNER'
-        uses: buildsville/add-remove-label@v2.0.1
+      - uses: actions/checkout@v3
         with:
-          token: ${{secrets.GITHUB_TOKEN}}
-          labels: needs-review
-          type: remove
+          # force to ci checkout main repo to avoid
+          # unexpected PR code to be executed with out github token
```

Comment:
The wording of this comment is a bit weird, maybe reword it to be more clear?

Reply:
It's actually an artifact from the old `labels.yaml` script that I didn't touch (and also appears in `lint-pr-title.yml`).  It is a little unclear, but I think the intent is to prevent the workflow from being executed against the head instead of base branch, which could allow someone to use the repo's secrets in newPR code, potentially exposing the token.  At least, I think that's the meaning?